### DISCUSSION TITLE
Does boogie have "memory function"?

### DISCUSSION THREAD

**Discussion by Luweicai** [boogie-org/boogie#473] [Q&A] [answered]
I might be crazy, but it seems that I come accross into the situation that when I use boogie to run one testcase a lot of times. (May be after a week)Boogie suddently become much more faster.

**Comment by shazqadeer** [comment, marked as answer]
I don't know what you mean by "memory function".  If you mean that the previously-computed results are cached on disk, Boogie does not do anything like that.
